src/tests/nativeaot/SmokeTests/HelloWasm/shell.js

```diff
@@ -0,0 +1,130 @@
+// No copyright header as this is largely a cut and paste from the standard
+// Emscripten produced shell
```

Comment:
Is all of the JS needed? It would be nice to trim it down to a more readable state.

Reply:
Have removed most of it.